test/cookbooks/php_ini-test/.gitignore

```diff
@@ -0,0 +1,21 @@
+.vagrant
+*~
+*#
+.#*
+\#*#
+.*.sw[a-z]
+*.un~
+
+# Bundler
+Gemfile.lock
+bin/*
+.bundle/*
+
+# test kitchen
+.kitchen/
+.kitchen.local.yml
+
+# Chef
+Berksfile.lock
+.zero-knife.rb
+Policyfile.lock.json
```

Comment:
Test cookbooks need to be very minimal and you don't need to use the code generator to make it. Please remove all of these extra files except for ``metadata.rb`` the recipes and any other files/templates you're using.